daemon/main.c

```diff
@@ -899,12 +899,9 @@ int main(int argc, char **argv) {
                             return 0;
                         }
                         else if(strncmp(optarg, createdataset_string, strlen(createdataset_string)) == 0) {
-                            unsigned history_seconds;
-
                             optarg += strlen(createdataset_string);
-                            history_seconds = (unsigned )strtoull(optarg, NULL, 0);
-
 #ifdef ENABLE_DBENGINE
+                            unsigned history_seconds = (unsigned )strtoull(optarg, NULL, 0);
```

Comment:
minor: extra space

> (unsigned )

Reply:
wait, how did this line get in the #ifdef? what's going on here?

Reply:
and why did you move the decleration together with the definition, this is not ANSI C, which it was before.